content/resources/help-centre/software-tips/command-line.md

```diff
@@ -0,0 +1,39 @@
+---
+title: The command line
+---
+
+## What is it?
+
+The command line, or command-line interface (CLI), is where commands or
+instructions can be passed from you, the user, to your computer. Many things
+that you do on your computer by clicking on a graphical user interface (GUI),
+can also be done in a CLI. For example, creating a new folder, or creating a
+text file. 
+
+The CLI is also known as a **CLI shell**. Sometimes simply referred to as a
+**shell**. A shell is essentially a program that we open on our computer, in
+order to type commands that we want the computer to execute. There are different
+'shells' available, depending on your operating system. You have probably seen
+or heard about `cmd.exe` or command prompt, which is a type of shell which comes
+with Windows by default. But there are other shell programs we can download and
+use, which have some added functionalities. For Windows we recommend downloading
+and using the following:
+
+- [Windows 
+    Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=en-au&gl=au)
```

Comment:
we should strongly recommend windows terminal - tabs, speed, zooming, themes, profiles, split panes, accuracy, colors, and most importantly emoji

Reply:
i sent you screen shots that show some differences